tests/test_management_api.py

```diff
@@ -99,6 +101,93 @@ def test_get_policy_api(self):
         self.assertTrue(e.has_grouping_policy(["alice", "data2_admin"]))
         self.assertFalse(e.has_grouping_policy(["bob", "data2_admin"]))
 
+    def test_get_policy_matching_function(self):
+        e = self.get_enforcer(
+            get_examples("rbac_with_domain_and_policy_pattern_model.conf"),
+            get_examples("rbac_with_domain_and_policy_pattern_policy.csv"),
+        )
+        e.get_role_manager().add_domain_matching_func(casbin.util.key_match2_func)
```

Comment:
This test case works without adding a domain matching function.

Reply:
Answer is below